hw/bsp/fomu/dfu.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/python
+
+# Written by Antonio Galea - 2010/11/18
+# Updated for DFU 1.1 by Sean Cross - 2020/03/31
+# Distributed under Gnu LGPL 3.0
+# see http://www.gnu.org/licenses/lgpl-3.0.txt
+
+import sys,struct,zlib,os
+from optparse import OptionParser
+
+DEFAULT_DEVICE="0x0483:0xdf11"
```

Comment:
This is the VID and PID for ArduPilot. Since this is in the `Fomu` directory, maybe this should be updated to `0x1209:0x5bf0`

Reply:
Although leaving it as-is does give an idea of some of the lineage of this code!

Reply:
updated, I just copy these from the circuitpython https://github.com/adafruit/circuitpython/blob/master/tools/dfu.py for an way to covert the bin to dfu file